.github/workflows/mobile-test-run.yml

```diff
@@ -69,6 +77,7 @@ jobs:
         ./gradlew runStories -Pvividus.configuration.suites=mobile_app/ios \
                              -Pvividus.configuration.profiles=mobile_app/local,mobile_app/ios \
                              -Pvividus.allure.history-directory=output/history/ios \
+                             -Pvividus.selenium.grid.url=http://127.0.0.1:4723/ \
```

Comment:
I propose to use --base-path /wd/hub option  to keep using mobile_app/local profile? 
Once we migrate VIVIDUS to 2-0 we can delete this option and update the profule